packages/zowe-explorer/src/trees/dataset/DatasetTree.ts

```diff
@@ -112,16 +112,26 @@ export class DatasetTree extends ZoweTreeProvider<IZoweDatasetTreeNode> implemen
         recursiveCall?: boolean
     ): Promise<void> {
         const destinationInfo = FsAbstractUtils.getInfoForUri(destUri, Profiles.getInstance());
-        let dsname = destUri.path.substring(destinationInfo.slashAfterProfilePos);
+        const sourceInfo = FsAbstractUtils.getInfoForUri(sourceUri, Profiles.getInstance());
+        let dsname = destUri.path.substring(destinationInfo.slashAfterProfilePos + 1);
         if (SharedContext.isPds(sourceNode)) {
             if (!DatasetFSProvider.instance.exists(destUri)) {
-                // create a PDS on remote
+                // find source PDS attributes and make destination PDS have same attributes
+                const sourceAttributesResponse = await ZoweExplorerApiRegister.getMvsApi(sourceInfo.profile).dataSet(dsname, {
+                    attributes: true,
+                    responseTimeout: sourceInfo.profile?.profile?.responseTimeout,
+                });
+                const { dsname: dsnameSource, ...rest } = sourceAttributesResponse.apiResponse.items[0];
+
+                // create a PDS on remote with the same attributes as source PDS
+                const transformedAttrs = (zosfiles.Copy as any).dataSetCrossLPAR({}, rest);
```

Comment:
`dataSetCrossLPAR` is public, the cast `as any` is not needed and ignores type checks:

 
```suggestion
                const transformedAttrs = zosfiles.Copy.dataSetCrossLPAR({}, rest);
```

Note that you may need to update the parameters once removing the `as any` cast, as there are 5 required parameters for the function and only 2 are provided here

Reply:
While this comment _is_ valid, I would advise not calling this function here because we already have logic that performs the cross-LPAR copy, so this would result in a duplicate copy attempt.

To clarify, I think `generateDatasetOptions` is the right function to use here - since its private, we'd have to either make the function public in the SDK, or keep the `zosfiles.Copy as any` cast in place. From there, we can pass those options to the `createDataSet` API like what was done in https://github.com/zowe/zowe-explorer-vscode/pull/3731/commits/d4929c17a064f698d6357e49f9459f9be779eda4.